test/src/mistralai_client_dart_base_chat_stream_test.dart

```diff
@@ -82,7 +82,6 @@ void main() {
               'role': 'role1',
               'content': 'content1',
               'name': 'name1',
-              'tool_calls': null,
```

Comment:
I think we also need to add test just for checking if there is any null in the json. If there is then we have a bug.